app/views/records/edit_fields/_event_date.html.erb

```diff
@@ -1,3 +1,44 @@
+<% template = f.object.model.class.to_s.underscore %>
 <% event_date = f.object[:event_date] %>
-<%= render 'hyku_addons/event_date/event_date', f: f, event_date: event_date %>
-<%= render 'hyku_addons/event_date/event_date_js' %>
+<% # The original file used .to_s instead of .first, but this returned an array with a Triple instance %>
+<% date_value = f.object.model.send("event_date").first.split('-') %>
```

Comment:
@cjcolvar This was previously `f.object.model.send("event_date").to_s` but that seemed to return an array with a single Triple instance in it and then nil whe `split` was called. I'm not sure if this is a consequence of the changes we are making?